Changed format of debug messages so that rosconsole_bridge can correctly parse the prefix

davetcoleman opened the pull request:
Reduces a ton of noise in moveit_core, for use with https://github.com/ros/rosconsole_bridge/pull/4

Also, my emacs automatically removed trailing whitespace...

tfoote commented on src/class_loader_core.cpp:
```diff
@@ -415,65 +415,59 @@ void purgeGraveyardOfMetaobjects(const std::string& library_path, ClassLoader* l
+  }
+  catch(const Poco::NotFoundException& e)
   {
-    boost::recursive_mutex::scoped_lock loader_lock(loader_mutex);
```
This removes a lock not mentioned in the comments. What that intended?

tfoote commented:
The mixing of whitespace and changes are very hard to read. Could you split it at least into two commits. You should be able to checkout the old version. Fix the whitespace, commit it. And then rebase your change on top of the cleaned up version. Then we can look at the two commits separately.

davetcoleman committed:
Improved debug output

Fixed find-replace-all mistake

Fixed more log messages to new format

Fixed more log messages to new format

davetcoleman commented:
Yes, I added that feature to my emacs today then immediately regretted it today. I just turned it off. Because of that, I had missed that deletion of the mutex, really good catch. 

I just restarted and made my changes again, the pull request is now clean. Thanks!

dirk-thomas commented on src/multi_library_class_loader.cpp:
```diff
@@ -75,7 +75,7 @@ bool MultiLibraryClassLoader::isLibraryAvailable(const std::string& library_name
 {
   if(!isLibraryAvailable(library_path))
-    active_class_loaders_[library_path] = new class_loader::ClassLoader(library_path, isOnDemandLoadUnloadEnabled());
+    active_class_loaders_[library_path] = new class_loader.ClassLoader(library_path, isOnDemandLoadUnloadEnabled());
```
I guess this change is not intended?

davetcoleman replied:
That's embarrassing! When I refactored the pull request I forgot to compile again :/
Fixed.

isucan commented:
+1
